schemas/pulse-job.yml

```diff
@@ -116,9 +124,11 @@ properties:
       - unknown
   jobKind:
     type: "string"
+    default: "other"
     enum:
       - build
       - test
+      - other
   tier:
```

Comment:
I'd rather stay with just an ``integer`` for now.  If we need a 2.5 or something, in the future, we can change it from ``integer`` to ``float`` (``number``) easy enough